### PR TITLE
Fix for RestOpenApiProcessor for parsing path variables from the http request uri succesfully. 

### PR DESCRIPTION
**Pull request by egekaraosmanoglu**
Fix/Update org.apache.camel.component.rest.openapi.RestOpenApiProcessor.java

When RestOpenApiProcessor tries to process the path variables in the request,  HttpHelper.evalPlaceholders  throws an ArrayIndexOutOfBoundsException because consumerPath  from the openApi spec starts with a "/" but the uri of the request does not start with a "/".

# Target

- [ ] I checked that the commit is targeting the correct branch (note that Camel 3 uses `camel-3.x`, whereas Camel 4 uses the `main` branch)

# Tracking
- [ ] If this is a large change, bug fix, or code improvement, I checked there is a [JIRA issue](https://issues.apache.org/jira/browse/CAMEL) filed for the change (usually before you start working on it).

# Apache Camel coding standards and style

- [ ] I checked that each commit in the pull request has a meaningful subject line and body.

- [ ] I have run `mvn clean install -DskipTests` locally from root folder and I have committed all auto-generated changes.

### CODE REVIEW

File: components/camel-rest-openapi/src/main/java/org/apache/camel/component/rest/openapi/RestOpenApiProcessor.java [hunk 1]
```diff
@@ -109,6 +109,14 @@ public boolean process(Exchange exchange, AsyncCallback callback) {
         if (m instanceof RestOpenApiConsumerPath rcp) {
             Operation o = rcp.getConsumer();
 
+            String consumerPath = rcp.getConsumerPath();
+
+            //if uri is not starting with slash then remove the slash in the consumerPath from the openApi spec
+            if(consumerPath.startsWith("/") && !uri.startsWith("/"))
```
Comment (davsclaus) [inline components/camel-rest-openapi/src/main/java/org/apache/camel/component/rest/openapi/RestOpenApiProcessor.java:115]: Add space after if

Comment (davsclaus) [reply components/camel-rest-openapi/src/main/java/org/apache/camel/component/rest/openapi/RestOpenApiProcessor.java:115]: And use same code style as elsewhere, ie curly bracket on same line etc

**Review by davsclaus — CHANGES_REQUESTED** [review]
code style changes needed for PR to build

File: components/camel-rest-openapi/src/main/java/org/apache/camel/component/rest/openapi/RestOpenApiProcessor.java (hunk 1 above)
Comment (egekaraosmanoglu) [reply components/camel-rest-openapi/src/main/java/org/apache/camel/component/rest/openapi/RestOpenApiProcessor.java:115]: sure thanks, also I see that I forgot to use the variable  consumerPath while calling HttpHelper.evalPlaceholders.

**Comment by egekaraosmanoglu** [comment]
Hi, applied the code style fixes and added a null check. I hope it is fine now! Thanks.

**Review by davsclaus — APPROVED** [review]

**Pull request merged**